src/silx/gui/plot/items/marker.py

```diff
@@ -141,6 +146,38 @@ def setFont(self, font: qt.QFont | None):
             self._font = None if font is None else qt.QFont(font)
             self._updated(ItemChangedType.FONT)
 
+    def getBackgroundColor(self) -> colors.RGBAColorType | numpy.ndarray | None:
+        """Returns the RGBA background color of the item
+
+        :rtype: 4-tuple of float in [0, 1] or array of colors
+        """
+        return self._bgColor
+
+    def setBackgroundColor(self, color, copy=True):
+        """Set item background  color
+
+        :param color: color(s) to be used
+        :type color: str ("#RRGGBB") or (npoints, 4) unsigned byte array or
+                     one of the predefined color names defined in colors.py
+        :param bool copy: True (Default) to get a copy,
+                         False to use internal representation (do not modify!)
+        """
+        if color is None:
+            pass
+        elif isinstance(color, (str, qt.QColor)):
+            color = colors.rgba(color)
+        else:
+            color = numpy.array(color, copy=copy)
+            # TODO more checks + improve color array support
+            if color.ndim == 1:  # Single RGBA color
+                color = colors.rgba(color)
+            else:  # Array of colors
+                assert color.ndim == 2
+
+        if self._bgColor != color:
+            self._bgColor = color
+            self._updated(ItemChangedType.BACKGROUND_COLOR)
```

Comment:
Removing support of array of colors:
```suggestion
    def setBackgroundColor(self, color: colors.ColorType):
        """Set item background  color

        :param color:
            color to be used as a str ("#RRGGBB") or (npoints, 4) unsigned byte array or
            one of the predefined color names defined in colors.py
        """
        if color is not None:
            color = colors.rgba(color)

        if self._bgColor != color:
            self._bgColor = color
            self._updated(ItemChangedType.BACKGROUND_COLOR)
```